Makefile

```diff
@@ -80,24 +82,35 @@ gen-swagger:
 		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
 		-i /local/${SPEC_PATCHED_FILE}
 
-gen: gen-swagger
-
-validate:
-	${VALIDATE} validate \
-		--recommend \
+# Requires downloading the specific version of the swagger-codegen-cli.jar from the artifact website
+# and putting it in the root of the repository. It will be .gitignore'd but it needs to be present.
+# Command below matches the Docker image Swagger Codegen CLI version above 3.0.34
+# curl -o swagger-codegen-cli.jar https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.34/swagger-codegen-cli-3.0.34.jar
+gen-swagger-apple-chip-cpu:
+	java -jar swagger-codegen-cli.jar generate -l go \
+		--additional-properties packageName=${PACKAGE_MAJOR} \
+		--model-package types \
+		--api-package models \
+		--git-user-id ${GIT_ORG} \
+		--git-repo-id ${GIT_REPO} \
+		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
 		-i /local/${SPEC_PATCHED_FILE}
 
-spectral:
```

Comment:
I have no concerns about removing this task (it doesn't seem to be used anywhere, and SDK generation is almost certainly too late in the API / spec release process to be looking for issues), but I'm curious if this was _ever_ used, and if so how.

Reply:
@displague added it on July 6, 2022 according to the git blame. I imagine that it was probably to check for issues because there may not have been a linter around at that time? Not sure what the history of OAS linting was within Fabric when these projects were born.

Reply:
https://github.com/equinix-labs/fabric-go/pull/2

Looks like I may have been drawing attention to spec concerns that were not being addressed through other linter processes. If yanking this, grab the .spectral.json file too.

It would be interesting to see what the output is against the current spec.